pkg/skaffold/deploy/kubectl.go

```diff
@@ -121,6 +133,34 @@ func (k *KubectlDeployer) manifestFiles(manifests []string) ([]string, error) {
 	return filteredManifests, nil
 }
 
+func parseManifestsForDeploys(manifests manifestList) ([]Artifact, error) {
+	results := []Artifact{}
+	for _, manifest := range manifests {
+		b := bufio.NewReader(bytes.NewReader(manifest))
```

Comment:
Why do you wrap this in bufio?

Reply:
the YAMLReader from apimachinery requires a bufio.Reader instead of an io.Reader: https://github.com/kubernetes/apimachinery/blob/master/pkg/util/yaml/decoder.go#L262